src/site/content/en/identity/passkey-form-autofill/index.md

```diff
@@ -18,168 +18,269 @@ tags:
 [Passkeys](https://developers.google.com/identity/passkeys) replace passwords
 and make user accounts on the web safer, simpler, easier to use. However, the
 transition from password-based to passkey-based authentication can complicate
-the user experience. Using form autofill to suggest passkeys can help
+the user experience. Leveraging form autofill to suggest passkeys can help
```

Comment:
Personally, I prefer use over leverage—is there any specific reason to change this phrasing?

Reply:
```suggestion
the user experience. Using form autofill to suggest passkeys can help
```